update.sh

```diff
@@ -0,0 +1 @@
+git submodule update --init --remote --no-single-branch --depth 1
```

Comment:
I would add `#!/bin/bash` at the start to be safe

Also add `git pull` before the submodule call so we have the latest content from the repo if this has been checked out previously